README.md

```diff
@@ -18,6 +18,7 @@ This module provides Stackdriver Trace support for Node.js applications. [Stackd
 1. Your application will need to be using Node.js version 4.0 or greater.
 1. You will need a project in the [Google Developers Console][cloud-console]. Your application can run anywhere, but the trace data is associated with a particular project.
 1. [Enable the Trace API](https://console.cloud.google.com/flows/enableapi?apiid=cloudtrace) for your project.
+1. At this point, code using async/await is not supported by this module. See [this section](#how-does-automatic-tracing-work).
```

Comment:
I am not sure if I want to be too wordy up here. If it is too wordy, we should skip this from here and let the details section explain this out.